client/src/main/java/org/apache/uniffle/client/util/ClientUtils.java

```diff
@@ -62,7 +62,7 @@ public static RemoteStorageInfo fetchRemoteStorage(
       String storageType,
       ShuffleWriteClient shuffleWriteClient) {
     RemoteStorageInfo remoteStorage = defaultRemoteStorage;
-    if (requireRemoteStorage(storageType)) {
+    if (StorageType.withRemoteStorage(StorageType.valueOf(storageType))) {
```

Comment:
Check `storageType != null` here?
The test fails because `storageType` is `null`.